en/advanced_config/tuning_the_ecl_ekf.md

```diff
@@ -431,30 +431,17 @@ A good tuning is obtained as follows:
 
 ### External Vision System
 
-Position, velocity or orientation measurements from an external vision system, e.g. Vicon, can be used:
-
-* External vision system horizontal position data will be used if bit position 3 in the [EKF2_AID_MASK](../advanced_config/parameter_reference.md#EKF2_AID_MASK) parameter is true.
-* External vision system vertical position data will be used if the [EKF2_HGT_REF](../advanced_config/parameter_reference.md#EKF2_HGT_REF) parameter is set to 3.
-  For more details about the configuration of height sources, [click here](#height).
-* External vision system velocity data will be used if bit position 8 in the [EKF2_AID_MASK](../advanced_config/parameter_reference.md#EKF2_AID_MASK) parameter is true.
-* External vision system orientation data will be used for yaw estimation if bit position 4 in the [EKF2_AID_MASK](../advanced_config/parameter_reference.md#EKF2_AID_MASK) parameter is true.
-* External vision reference frame offset will be estimated and used to rotate the external vision system data if bit position 6 in the [EKF2_AID_MASK](../advanced_config/parameter_reference.md#EKF2_AID_MASK) parameter is true.
-
-Either bit 4 (`EV_YAW`) or bit 6 (`EV_ROTATE`) should be set to true, but not both together.
-Following [EKF2_AID_MASK](../advanced_config/parameter_reference.md#EKF2_AID_MASK) values are supported when using with an external vision system.
-
-EKF_AID_MASK value | Set bits | Description
---- | --- | ---
-320 | EV_VEL + ROTATE_EV | Heading w.r.t. North
-24 | EV_POS + EV_YAW | Heading w.r.t. external vision frame
-72 | EV_POS + ROTATE_EV | Heading w.r.t. North
-272 | EV_VEL + EV_YAW | Heading w.r.t. external vision frame
-280 | EV_POS + EV_VEL + EV_YAW | Heading w.r.t. external vision frame
-328 | EV_POS + EV_VEL + ROTATE_EV | Heading w.r.t. North
+Position, velocity or orientation measurements from an external vision system, e.g. Vicon, can be used.
 
-:::tip
-When using external vision in combination with [GNSS fusion](#gps), it is recommended to use `EV_VEL` and not `EV_POS` in order to prevent the two position-fixing sources from fighting against each other.
-:::
+The measurements that are fused are configured by setting the appropriate bits of [EKF2_EV_CTRL](../advanced_config/parameter_reference.md#EKF2_EV_CTRL) to `true`:
+
+- `0`: Horizontal position data 
+- `1`: Vertical position data.
+  Height sources may additionally be configured using [EKF2_HGT_REF](../advanced_config/parameter_reference.md#EKF2_HGT_REF) (see section [Height](#height)).
+- `2`: Velocity data 
+- `3`: Yaw data
+
+Note that if yaw data is used (bit 3) the heading is with respect to the external vision frame; otherwise the heading is relative to North.
```

Comment:
@hamishwillee as a future note, right now it should be possible to transit from indoor (EV) to outdoor (GPS) and vice versa i.e. have both GPS an EV potentially enabled at the same time, which I think it should be mentioned in the documentation.
I'm still trying to figure out how it works though.

Reply:
That's cool, but not until we know how. Can I leave that with you for now?

Reply:
I would suggest that we don't mention it in the docs until we're really sure that this works as expected.